neo/tools/compilers/fontcodec/FontTool.cxx

```diff
@@ -0,0 +1,40 @@
+/*
+ * FontTool.cxx
```

Comment:
Pls rename the file to FontTool.cpp and adjust the include in cmake according to this, we should stay consistent in how we name source and header files.